### ISSUE TITLE
Mystical Agriculture Inferium crops seeds incorrect

### ISSUE THREAD

**Issue by Xavion3** [AgriCraft/AgriPlants#18]
In Mystical Agriculture the unlocalized names for the tiered inferium seeds are just as `tier1_inferium_seeds`, seen [here](https://github.com/BlakeBr0/MysticalAgriculture/blob/1ae78e1ed5239bb1d0c78b1102cb507c282f8d8d/src/main/java/com/blakebr0/mysticalagriculture/items/ModItems.java#L159-L163), so the incorrect names such as in [here](https://github.com/AgriCraft/AgriPlants/blob/master/mod_mysticalagriculture/plants/tier1_inferium_plant.json#L9) and the four similar files will cause the seeds to be unable to be placed on cropsticks.

Beyond that, all five tiers appear to have exactly the same drops, while in Mystical Agriculture they normally drop their tier in essence, not sure how the drop counts and the max/min stuff work though which is part of why this is an issue and not a pull request, that and I'm not sure how to best translate that into the agricraft gain dependent model. But normally tier 1 would drop 1 essence, tier 3 would drop 3 essence, and so on.

**Comment by TokkenGmr** [comment]
While I was creating my custom pack and seeds for agricraft, I looked at the essence seeds and then thought about tier 1 with 10/10/10 and was like there is no real need for any other tier. So in my custom pack there is only one essence seed. 

With that being said I am guessing the best change would be just to make the min drop match the tier, however tier 5 10/10/10 may give a lil to much but thats my thoughts. I was going to be adding the new seeds that were added by mystical agriculture so I can make these changes as well
